### ISSUE TITLE
Forbid yield-from that can be replaced by yield without cycle

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#953]
# Rule request

## Thesis

Forbid yield-from that can be replaced by yield without cycle and in less than 3 lines.

```python
# bad
yield from [value]
# good
yield value

# bad
yield from [v1, v2]
# good
yield v1
yield v2
```

Also bad:

```python
yield v1
yield v2
yield v3
yield v4
```

It's only about explicit lists. It's not about calling other generators, yielding from a variable, long lists etc.

```python
# ok
yield from something_else()
yield from (1, 2, 3, 4, 5, 6, 7, 8, 9)
```

Statements:

1. Let's enforce `yield from` to use `tuple` as a source. Not lists, not sets, but tuples
2. Let's enforce that we prefer `yield from` over individual repeated `yields` in case we have more than 2 consecutive items
3. We enforce `yield` over `yield from` with list with one or two items
4. We should also disallow all nodes that are used with `yield from` except `Tuple`, `GeneratorExpr`, `Call`, `Subscript`, `Attribute`, `Name`

## Reasoning

Consistency, keeping things as simple as possible.

**Comment by sobolevn** [comment]
I agree with this rule. But:

```python
# bad
yield [v1, v2]
# good
yield v1
yield v2
```

It is not the same thing:

```python
>>> def one():
...     yield [1, 2]
... 
>>> list(one())
[[1, 2]]
>>> def two():
...     yield 1
...     yield 2
... 
>>> list(two())
[1, 2]
```

**Comment by orsinium** [comment]
Sure, that's just a typo. Fixed.

**Comment by sobolevn** [comment]
Let's merge it with #893

**Comment by sobolevn** [comment]
Or not.

**Comment by sobolevn** [comment]
I have changed this rule a little.

Since I don't like inconsistency I have enforced to write `yield from` when `yield` is used at least two times consecutively. Once is fine, two need to be refactored into `yield from`.
